ARCs/arc-0036.md

```diff
@@ -0,0 +1,102 @@
+---
+arc: 36
+title: Convention for declaring filters of an NFT
+description: This is a convention for declaring filters in an NFT metadata
+author: Stéphane Barroso (@sudoweezy)
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/163
+status: Draft
+type: Standards Track
+category: ARC
+created: 2022-01-04
+requires: 16
+---
+
+# Standard for declaring filters inside non-fungible token metadata
+
+## Abstract
+
+The goal is to establish a standard for how filters are declared inside a non-fungible (NFT) metadata.
+
+## Specification
+
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+
+> Comments like this are non-normative.
+
+If the property `filters` is provided anywhere in the metadata of an nft, it **MUST** adhere to the schema below.
+If the nft is a part of a larger collection and that collection has filters, all the available filters for the collection **MUST** be listed as a property of the `filters` object.
+If the nft does not have a particular filter, it's value **MUST** be "none".
+
+The JSON schema for `filters` is as follows:
+
+```json
+{
+  "title": "Filters for Non-Fungible Token",
+  "type": "object",
+  "properties": {
+    "filters": {
+      "type": "array",
+      "description": "Filters can be used to filter nfts of a collection.  Values must be an array of filter."
+    }
+  }
+}
+```
+
+The JSON schema for a `filter` is as follows:
+```json
+{
+  "filter": {
+    "type": "object",
+    "description": "Filter (attributes) that can be used to identify an nft. Values may be strings or numbers"
+  }
+}
+```
+
+#### Examples
+
+##### Example of an NFT that has traits & filters
+
+```json
+{
+  "name": "NFT With Traits & filters",
+  "description": "NFT with traits & filters",
+  "image": "https://s3.amazonaws.com/your-bucket/images/two.png",
+  "image_integrity": "sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=",
+  "properties": {
+    "creator": "Tim Smith",
+    "created_at": "January 2, 2022",
+    "traits": {
+      "background": "yellow",
+      "head": "curly"
+    },
+    "filters": [
+      {
+        "type": "range",
```

Comment:
Should we define possible values for `type`? 

Maybe `numeric` instead of `range` given UIs filter numeric attributes in multiple ways. For example

<img width="829" alt="Screenshot 2023-03-14 at 10 30 42" src="https://user-images.githubusercontent.com/96784685/224973516-10f87a4d-c545-400a-89c0-162a16433cfd.png">